coco/state.py

```diff
@@ -165,18 +178,56 @@ def read_from_file(self, path, file):
         file : str
             Name of the file to read from.
         """
-        logger.debug(f"Loading state {path} from file {file}.")
+        logger.debug(f"Loading file {file} into state path '{path}'.")
 
         # Update persistent state
         with self._storage.update():
-            element, name = self._find_new(path)
+            if len(path) != 0:
```

Comment:
When would someone run

`read_from_file(path='')`?

Reply:
The new (in this PR) internal endpoint `load-state` does that.

Reply:
Rick explained in private that a `''` state refers to the root state.